### ISSUE TITLE
Undefined method join for nil:NilClass - (tags.join)

### ISSUE THREAD

**Issue by tylergohl** [DataDog/chef-datadog#254]
I'm getting the following on a Server 2008 R2 node while it's attempting to install the dd agent.  My guess is the `TAGS="#{node['tags'].join(',')` line is failing because my server likely has no tags.

```
 ================================================================================
 Recipe Compile Error in c:/chef/cache/cookbooks/datadog/recipes/dd-agent.rb
 ================================================================================

 NoMethodError
 -------------
 undefined method `join' for nil:NilClass

 Cookbook Trace:
 ---------------
   c:/chef/cache/cookbooks/datadog/recipes/_install-windows.rb:38:in `block in from_file'
   c:/chef/cache/cookbooks/datadog/recipes/_install-windows.rb:36:in `from_file'
   c:/chef/cache/cookbooks/datadog/recipes/dd-agent.rb:22:in `from_file'

 Relevant File Content:
 ----------------------
 c:/chef/cache/cookbooks/datadog/recipes/_install-windows.rb:

  31:    # As a workaround uninstall the package first if a new MSI is downloaded
  32:    notifies :remove, 'windows_package[Datadog Agent]', :immediately
  33:  end
  34:
  35:  # Install the package
  36:  windows_package 'Datadog Agent' do
  37:    source temp_file
  38>>   options %(APIKEY="#{node['datadog']['api_key']}" HOSTNAME="#{node['hostname']}" TAGS="#{node['tags'].join(',')
")
  39:    action :install
  40:  end
  41:
```

**Comment by tylergohl** [comment]
Confirmed. I added 2 chef tags to my server and the agent installed without issue.

**Comment by olivielpeau** [comment]
Thanks for reporting this issue @tylergohl!

On top of this bug, all the options that are currently passed to the MSI are actually useless because they are written to the `datadog.conf` file and this file gets overwritten later in the `dd-agent` recipe: https://github.com/DataDog/chef-datadog/blob/6ced7e69a468b14238c67d5b9258d30217a4be0d/recipes/dd-agent.rb#L52

#252 will fix this issue.

**Comment by miketheman** [comment]
Resolved in #286 for now. Closing this issue, will work towards fuller solution in #252 regardless.
